launchpad-starter-next-app/src/app/api/deploy/route.ts

```diff
@@ -15,26 +15,28 @@ export async function POST(request: Request) {
             );
         }
 
-        // 1. Spin up TEE simulator container if not already running
+        // 1. Start TEE container if not already running
         if (!containerManager.isRunning()) {
-            console.log("Starting TEE simulator container...");
+            console.log("Starting TEE container...");
             await containerManager.startContainer();
         }
 
-        // 2. Generate agent keys in TEE
-        const agentKeys = await containerManager.generateAgentKeys();
-        const agentPublicKey = agentKeys.keyAddress;
+        // 2. TODO get agent key from deployed TEE instance
```

Comment:
Then here we call the deploymentUrl/api/getPublicKey to get the agent key

Note: this api must be implemented in the docker image